### DISCUSSION TITLE
código java de misioneros y caníbales

### DISCUSSION THREAD

**Discussion by Adrw2019** [community/community#135462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

alguien tiene el código de java orientada a objetos del problema de los misioneros y caníbales

**Comment by davevad93** [comment]
Hi @Adrw2019 ,

Take a look at this repo, I think you may find it useful.

- [Solucion-Problema-de-Misioneros-y-canibales](https://github.com/bidbiri/Solucion-Problema-de-Misioneros-y-canibales/blob/master/MisionerosYCanibales.java)

**Reply by Adrw2019** [reply]
muchas gracias

**Reply by davevad93** [reply]
You're welcome @Adrw2019 ! 👍

**Comment by Adrw2019** [comment]
Este es el resultado que aparece en consola: 

Misioneros:3,Canibales:2,Bote:1____Misioneros:0,Canibales:0,Bote:0
Numero de personas sin trasladar:5
Misioneros:3,Canibales:0,Bote:0____Misioneros:0,Canibales:2,Bote:1
Numero de personas sin trasladar:3
Misioneros:3,Canibales:1,Bote:1____Misioneros:0,Canibales:1,Bote:0
Numero de personas sin trasladar:4
Misioneros:3,Canibales:1,Bote:0____Misioneros:0,Canibales:3,Bote:1
Numero de personas sin trasladar:2
Misioneros:3,Canibales:0,Bote:1____Misioneros:0,Canibales:2,Bote:0
Numero de personas sin trasladar:3
Misioneros:1,Canibales:0,Bote:0____Misioneros:2,Canibales:2,Bote:1
Numero de personas sin trasladar:1
Misioneros:2,Canibales:1,Bote:1____Misioneros:1,Canibales:1,Bote:0
Numero de personas sin trasladar:3
Misioneros:0,Canibales:1,Bote:0____Misioneros:3,Canibales:1,Bote:1
Numero de personas sin trasladar:1
Misioneros:0,Canibales:2,Bote:1____Misioneros:3,Canibales:0,Bote:0
Numero de personas sin trasladar:2
Misioneros:0,Canibales:0,Bote:0____Misioneros:3,Canibales:2,Bote:1
Numero de personas sin trasladar:0
Misioneros:0,Canibales:1,Bote:1____Misioneros:3,Canibales:1,Bote:0
Numero de personas sin trasladar:1
Misioneros:0,Canibales:1,Bote:0____Misioneros:3,Canibales:3,Bote:1
Numero de personas sin trasladar:-1
BUILD SUCCESSFUL (total time: 0 seconds)



Pero necesito agregar unas cosas, este sería el modelo de como necesito que se imprima en consola.

run:
Estado inicial lado izquierdo Misioneros:3,Canibales:3 y lado derecho Misioneros:0,Canibales:0.

Misioneros:3,Canibales:3,Bote:1____Misioneros:0,Canibales:0,Bote:0
Numero de personas sin trasladar:6
Misioneros:3,Canibales:2,Bote:1____Misioneros:0,Canibales:0,Bote:0
Numero de personas sin trasladar:5
Misioneros:3,Canibales:0,Bote:0____Misioneros:0,Canibales:2,Bote:1
Numero de personas sin trasladar:3
Misioneros:3,Canibales:1,Bote:1____Misioneros:0,Canibales:1,Bote:0
Numero de personas sin trasladar:4
Misioneros:3,Canibales:1,Bote:0____Misioneros:0,Canibales:3,Bote:1
Numero de personas sin trasladar:2
Misioneros:3,Canibales:0,Bote:1____Misioneros:0,Canibales:2,Bote:0
Numero de personas sin trasladar:3
Misioneros:1,Canibales:0,Bote:0____Misioneros:2,Canibales:2,Bote:1
Numero de personas sin trasladar:1
Misioneros:2,Canibales:1,Bote:1____Misioneros:1,Canibales:1,Bote:0
Numero de personas sin trasladar:3
Misioneros:0,Canibales:1,Bote:0____Misioneros:3,Canibales:1,Bote:1
Numero de personas sin trasladar:1
Misioneros:0,Canibales:2,Bote:1____Misioneros:3,Canibales:0,Bote:0
Numero de personas sin trasladar:2
Misioneros:0,Canibales:0,Bote:0____Misioneros:3,Canibales:2,Bote:1
Numero de personas sin trasladar:0
Misioneros:0,Canibales:1,Bote:1____Misioneros:3,Canibales:1,Bote:0
Numero de personas sin trasladar:1
Misioneros:0,Canibales:0,Bote:0____Misioneros:3,Canibales:3,Bote:1
Numero de personas sin trasladar:0

Estado final lado izquierdo Misioneros:0,Canibales:0 y lado derecho Misioneros:3,Canibales:3.


Pero necesito agregar unas cosas, este seria el modelo de como necesito que se imprima en consola.
